Source/Main.c

```diff
@@ -112,30 +113,34 @@ int main(void)
         DL_APPEND(periodic_message_list, periodic_message);
     }
 
-    server_start(port,
-                 64,
-                 2,
-                 0,
-                 0,
-                 master,
-                 map_list,
-                 map_list_len,
-                 welcome_message_list,
-                 welcome_message_list_len,
-                 periodic_message_list,
-                 periodic_message_list_len,
-                 periodicDelays,
-                 manager_passwd,
-                 admin_passwd,
-                 mod_passwd,
-                 guard_passwd,
-                 trusted_passwd,
-                 server_name,
-                 team1_name,
-                 team2_name,
-                 team1_color,
-                 team2_color,
-                 gamemode);
+    struct server_start_options options = {
+      port,
+      64,
+      2,
+      0,
+      0,
+      master,
+      map_list,
+      map_list_len,
+      welcome_message_list,
+      welcome_message_list_len,
+      periodic_message_list,
+      periodic_message_list_len,
+      periodicDelays,
+      manager_passwd,
+      admin_passwd,
+      mod_passwd,
+      guard_passwd,
+      trusted_passwd,
+      server_name,
+      team1_name,
+      team2_name,
+      team1_color,
+      team2_color,
+      gamemode
+    };
```

Comment:
would be nice to use named struct initialization syntax like:
```c
struct server_start_options options = {
.port = port,
.connections = 64,
...
```
though im not sure if it's really an objective improvement or will it become an annoyance over time... 🤔

Reply:
I was thinking about that. I will give it a try. I would personally like named initialization because otherwise it's as hard to read as the args. :P